doc/zero-to-binderhub/setup-registry.rst

```diff
@@ -137,6 +137,28 @@ where:
        SERVICE_PRINCIPAL_PASSWORD=$(az ad sp create-for-rbac --name <SP_NAME> --role AcrPush --scopes <ACR_ID> --query password --output tsv)
        SERVICE_PRINCIPAL_ID=$(az ad sp show --id http://<SP_NAME> --query appId --output tsv)
 
+Set up the OVH Container Registry
+---------------------------------
+
+To use the OVH Container Registry, log in to the `OVH Control Panel <https://www.ovh.com/manager/public-cloud>`_.
+
+1. Click on **Managed Private Registry**
+
+2. Click on the **Create a private registry** button
+
+3. Select a **Region**, a **Name** for the registry and a **Plan**.
+
+4. Generate the **Harbor identification details** to connect to the registry
+
+.. figure:: ovh/generate_id_details.png
+
+  Generate the identification details to connect to
+  the registry.
+
+5. Then click on **Harbor API** to retrieve the URL of the registry
+
+For more information about these steps, check out the `OVH Documentation <https://docs.ovh.com/gb/en/private-registry/creating-a-private-registry>`_
+
```

Comment:
At that point, it may be useful to prompt the user to connect to the Harbor UI (using the credentials they created earlier) and to create a new project, mentioning that the project identifier will be part of the prefix?

Reply:
Yes this step is missing, and might not be obvious at first.